boofuzz/__init__.py

```diff
@@ -1,37 +1,70 @@
 from __future__ import absolute_import
 
 import functools
+
 import six
+from past.builtins import map
 
-from . import blocks
-from . import legos
-from . import pedrpc
-from . import primitives
-from . import exception
-from .blocks import REQUESTS
-from .blocks.block import Block
-from .blocks.checksum import Checksum
-from .blocks.repeat import Repeat
-from .blocks.request import Request
-from .blocks.size import Size
-from .constants import BIG_ENDIAN, LITTLE_ENDIAN, DEFAULT_PROCMON_PORT
+from . import blocks, exception, legos, pedrpc, primitives
+from .blocks import Block, Checksum, Repeat, Request, REQUESTS, Size
+from .constants import BIG_ENDIAN, DEFAULT_PROCMON_PORT, LITTLE_ENDIAN
 from .event_hook import EventHook
+from .exception import MustImplementException, SizerNotUtilizedError, SullyRuntimeError
 from .fuzz_logger import FuzzLogger
 from .fuzz_logger_csv import FuzzLoggerCsv
-from .fuzz_logger_text import FuzzLoggerText
 from .fuzz_logger_curses import FuzzLoggerCurses
+from .fuzz_logger_text import FuzzLoggerText
 from .ifuzz_logger import IFuzzLogger
 from .ifuzz_logger_backend import IFuzzLoggerBackend
 from .itarget_connection import ITargetConnection
-from .primitives import (BasePrimitive, Delim, Group,
-                         RandomData, Static, String, BitField,
-                         Byte, Word, DWord, QWord, FromFile, Mirror)
+from .primitives import (BasePrimitive, BitField, Byte, Delim, DWord, FromFile, Group, Mirror, QWord, RandomData,
+                         Static, String, Word)
 from .serial_connection import SerialConnection
-from .sessions import Session, Target, open_test_run
-from .exception import SullyRuntimeError, SizerNotUtilizedError, MustImplementException
+from .sessions import open_test_run, Session, Target
 from .socket_connection import SocketConnection
-from past.builtins import map
-from builtins import chr
```

Comment:
Why do we import all submodules here? Do they all have to be callable independently without `boofuzz.`? Same question for other inits.

Reply:
The goal is similar to the goal described in the question here: https://stackoverflow.com/questions/31079047/python-pep8-class-in-init-imported-but-not-used

The idea is to enable `from boofuzz import Socketconnection` etc. instead of `from boofuzz.socket_connection import SocketConnection`.

Reply:
Makes sense, let's leave it as it is for now.